composer.json

```diff
@@ -26,16 +26,23 @@
     },
     "autoload": {
         "psr-4": {
-            "ShipMonk\\PHPStan\\": "src/"
+            "ShipMonk\\PHPStan\\": [
+                "src/lib/",
+                "src/dev/"
```

Comment:
Do we really want to load `dev` in regular autoload? I'd expect it to be relevant only for `autoload-dev`